src/httpclient.c

```diff
@@ -523,6 +566,9 @@ static void usage(const char *progname)
             "               verification\n"
             "  -X <local-udp-port>\n"
             "               specifies that the tunnel being created is a CONNECT-UDP tunnel\n"
+            "  --connect-udp-rfc-version <draft03 | rfc9298>\n"
```

Comment:
I still need to address your request to change this to a boolean flag.
* Would you prefer --connect-udp-draft03 or --connect-udp-rfc9298?
* What should be the default?

https://github.com/h2o/h2o/pull/3305#discussion_r1383474895

Reply:
I think we'd prefer `--connect-udp-draft03`. We are moving towards RFC 9298 only, that should preferably be the default.

Reply:
Done.